src/PrestaShopBundle/Service/Hook/HookEvent.php

```diff
@@ -54,25 +81,12 @@ public function setHookParameters($parameters)
     }
 
     /**
-     * Returns Hook parameters and default values.
+     * Returns Hook parameters and context parameters
      *
-     * More values than the param set is returned:
-     * - _ps_version contains PrestaShop version, and is here only if the Hook is triggered by Symfony architecture.
-     * These values can either be overriden by setHookParameters using the same parameter key.
-     *
-     * @return array the array of hook parameters, more default fixed values
+     * @return array
      */
     public function getHookParameters()
     {
-        $globalParameters = ['_ps_version' => AppKernel::VERSION];
-
-        $sfContainer = SymfonyContainer::getInstance();
```

Comment:
This is the horror Dependency Injection I wanted to clean

Reply:
Good job ;)